postgres/datadog_checks/postgres/statements.py

```diff
@@ -27,15 +28,21 @@
 except ImportError:
     from ..stubs import datadog_agent
 
+QUERYID_TO_CALLS_QUERY = """
+SELECT queryid, calls
+  FROM {pg_stat_statements_view}
+  WHERE queryid IS NOT NULL
+"""
+
 STATEMENTS_QUERY = """
 SELECT {cols}
   FROM {pg_stat_statements_view} as pg_stat_statements
   LEFT JOIN pg_roles
          ON pg_stat_statements.userid = pg_roles.oid
   LEFT JOIN pg_database
          ON pg_stat_statements.dbid = pg_database.oid
-  WHERE query != '<insufficient privilege>'
-  AND query NOT LIKE 'EXPLAIN %%'
+  WHERE query NOT LIKE 'EXPLAIN %%'
```

Comment:
why did we remove `query != '<insufficient privilege>'`?

Reply:
This is from the previous PR - since we're filtering based on `queryid`, the rows where the query text gets set to this don't have a queryid so don't get returned. But now that we're querying all of `pg_stat_statements` to get the initial baseline state, I'll add this back.

Reply:
Actually this takes care of that: https://github.com/DataDog/integrations-core/pull/17554/files#diff-157c2960ae5c7408275369d11207e5635cb0515cdc0b898396c4d043df08576dR34. So this can stay.